### PR TITLE
2 search by username

### LINKED ISSUE #2: Search by username
As a hiring partner
So I can find a specific user
I want to search for a particular user by username and view them in the list

### PR DESCRIPTION
**Pull request by drjparry**
Closes #2

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -1,43 +1,30 @@
...
-    <li><a href="#/view1">view1</a></li>
-    <li><a href="#/view2">view2</a></li>
-  </ul>
+<body ng-controller='userListCtrl'>
 
-  <!--[if lt IE 7]>
-      <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-  <![endif]-->
+  <input ng-model='query'>
+  <button id='button'ng-click='searchUser()'>Search</button>
```
Comment (sjmog) [inline app/index.html:15]: I don't like this ID - it's not very descriptive. What is this button?

File: app/js/controllers.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
+githubCreep.controller('userListCtrl', ['$scope', '$http',
+
```
Comment (sjmog) [inline app/js/controllers.js:3]: Weird space lines

File: app/js/controllers.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    $scope.searchUser=function(){
+      $http.get('https://api.github.com/search/users?q='+$scope.query).success(function(data) {
+      $scope.users = data.items;
+    });
+  };
+
+}])
+
+
+// .controller('gitHubDataController', ['$scope','$http', function($scope,$http) {
```
Comment (sjmog) [inline app/js/controllers.js:16]: Committing commented code :scream:

File: test/unit/controllersSpec.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    });
+
+
+  });
+});
+
+
+
+
+// 'use strict';
```
Comment (sjmog) [inline test/unit/controllersSpec.js:37]: Such screaming :scream_cat: - no need for commented code, it should all be visible in your commit history if people want it.

Comment (BjoernWagner) [inline test/unit/controllersSpec.js:37]: :shipit:

**Pull request merged**